perf_test

```diff
@@ -142,6 +162,24 @@ if __name__ == "__main__":
     if args.bandwidth + args.iops == []:
         args.bandwidth = ['1M']
 
+    if args.funccount:
+        if not shutil.which('funccount'):
+            sys.exit('Could not find funccount. Please install from https://github.com/brendangregg/perf-tools/pull/115')
+
+        if args.read:
+            print('funccount is only available for writes, it will be ignored for reads.')
+
+        if args.write:
+            try:
+                fc = Funccount(f'raid5_make_request raid5_write_one_full_stripe',
+                               sleep=args.ramptime)
+            except ValueError as e:
```

Comment:
Better to catch exceptions around all of the main code (which is getting quite large and could use factoring out into other functions).

You should also be catching subprocess errors and FileNotFoundErrors (which will indicate if funccount is not found or fio is not found) and handling them appropriately.